Jenkinsfile

```diff
@@ -442,6 +479,42 @@ try {
         }
       }
     },
+    'Python2: Quantize CPU': {
+      node('mxnetlinux-gpu-p3') {
```

Comment:
Possible c+p error.   Is it intended that cpu tests are running on a gpu node?

Reply:
I'm not familiar with this setup. Are we supposed not to run cpu tests on a gpu node? If that's the case, I would have to submit the CPU tests under `mxnetlinux-cpu`. Thanks.

Reply:
Exactly, generally if the test is cpu-only we'd run it on a 'mxnetlinux-cpu' node.

Reply:
I see. I will change that. Thanks for pointing it out.